mysql/seed/cscl_db.sql

```diff
@@ -753,7 +753,7 @@ INSERT INTO `Books` VALUES ('2253005274','Le Grand Meaulnes (Classiques De Poche
 INSERT INTO `Books` VALUES ('225300670X','A l\'Ouest rien de nouveau','Erich Maria Remarque',1973,'LGF','http://images.amazon.com/images/P/225300670X.01.THUMBZZZ.jpg','http://images.amazon.com/images/P/225300670X.01.MZZZZZZZ.jpg','http://images.amazon.com/images/P/225300670X.01.LZZZZZZZ.jpg');
 INSERT INTO `Books` VALUES ('2266084372','Pourquoi j\'ai mang�?© mon p�?¨re','Lewis',2000,'Pocket','http://images.amazon.com/images/P/2266084372.01.THUMBZZZ.jpg','http://images.amazon.com/images/P/2266084372.01.MZZZZZZZ.jpg','http://images.amazon.com/images/P/2266084372.01.LZZZZZZZ.jpg');
 INSERT INTO `Books` VALUES ('2070362388','Ravage','Rene Barjavel',0,'Gallimard French','http://images.amazon.com/images/P/2070362388.01.THUMBZZZ.jpg','http://images.amazon.com/images/P/2070362388.01.MZZZZZZZ.jpg','http://images.amazon.com/images/P/2070362388.01.LZZZZZZZ.jpg');
-INSERT INTO `Books` VALUES ('2266029460','Voyage d\'une parisienne �?  Lhassa','Alexandra David-N�?©el',1989,'Pocket','http://images.amazon.com/images/P/2266029460.01.THUMBZZZ.jpg','http://images.amazon.com/images/P/2266029460.01.MZZZZZZZ.jpg','http://images.amazon.com/images/P/2266029460.01.LZZZZZZZ.jpg');
+INSERT INTO `Books` VALUES ('2266029460','Voyage d\'une parisienne �?  Lhassa','Alexandra David-N�?©el',1989,'Pocket','http://images.amazon.com/images/P/2266029460.01.THUMBZZZ.jpg','http://images.amazon.com/images/P/2266029460.01.MZZZZZZZ.jpg','http://images.amazon.com/images/P/2266029460.01.LZZZZZZZ.jpg');

```

Comment:
@ovidio, @steve-louis - what's up with these special characters? If possible, maybe it'd be better to use a unicode value or something along those lines?